pkg/controller/statusmanager/status_manager.go

```diff
@@ -369,12 +369,26 @@ func (status *StatusManager) set(reachedAvailableLevel bool, conditions ...operv
 			)
 		}
 
-		v1helpers.SetOperatorCondition(&oc.Status.Conditions,
-			operv1.OperatorCondition{
-				Type:   operv1.OperatorStatusTypeUpgradeable,
-				Status: operv1.ConditionTrue,
-			},
-		)
+		if oc.Spec.DefaultNetwork.Type == operv1.NetworkTypeOpenShiftSDN {
+			// OpenShiftSDN is removed in 4.17, so block the upgrade if we have OpenShiftSDN in the spec.
+			v1helpers.SetOperatorCondition(&oc.Status.Conditions,
+				operv1.OperatorCondition{
+					Type:   string(configv1.OperatorUpgradeable),
```

Comment:
What's the reason for swapping `operv1.OperatorStatusTypeUpgradeable` with `string(configv1.OperatorUpgradeable)`?

Reply:
yes @kyrtapz, as we discussed offline, swapping is not needed. updated code to use `operv1.OperatorStatusTypeUpgradeable`.